src/content/docs/logs/get-started/logging-best-practices.mdx

```diff
@@ -0,0 +1,158 @@
+
+* To [search data partitions](/docs/logs/ui-data/data-partitions#search) in the **Logs** UI, you must select the appropriate partition(s), open the partition selector and check the partitions you want to search. If you're using NRQL, use the `FROM` clause to specify the `Log` or `Log_<partion>` to search. For example:
+
+ ```
+ FROM Log_<my_partition_name> SELECT * SINCE 1 hour ago
+ ```
+
+ Or to search logs on multiple partitions:
+
+ ```
+ FROM Log, Log_<my_partition_name> SELECT * SINCE 1 hour ago
+ ```
+
+## Parsing logs [#parsing-logs]
+
+Parsing your logs at ingest is the best way to make your log data more usable by you and other users in your organization.  When you parse out attributes, you can easily use them to search in the **Logs** UI and in NRQL without having to parse data at query time. This also allows you to use them easily in alerts and dashboards.
+
+For parsing logs we recommend that you:
+* Parse your logs at ingest to create `attributes` (or fields), which you can use when searching or creating dashboards and alerts. Attributes can be strings of data or numerical values.
+
+* Use the `logtype` attribute you added to your logs at ingest, along with other NRQL `WHERE` clauses to match the data you want to parse. Write specific matching rules to filter logs as precisely as possible. For example:
+
+ ```
+ WHERE logtype='mylog' AND message LIKE '%error%'
+ ```
+
+* Use our [built-in parsing rules](https://docs.newrelic.com/docs/logs/ui-data/built-log-parsing-rules/) and associated `logtype` attribute whenever possible. If the built-in rules don't work for your data, use a different `logtype` attribute name (i.e., `apache_logs` vs `apache`, `iis_w3c_custom` vs `iis_w3c`), and then create a new parsing rule in the UI using a modified version of the built-in rules so that it works for your log data format.
+
+* Use our **Parsing** UI to test and validate your Grok rules. Using the `Paste log` option, you can paste in one of your log messages to test your Grok expression before creating and saving a permanent parsing rule. 
+
+ <img
+   title="Log parsing example"
+   alt="Example of using Parsing UI to test Grok rule"
+   src={parsingExample}
+ />
+
+* Use external FluentBit configurations for parsing multi-line logs and for other, more extensive pre-parsing before ingesting into New Relic. For details and configuration of multi-line parsing with our infrastructure agent, see [this blog post](https://newrelic.com/blog/how-to-relic/parse-multiline-log-messages-fluent-bit-plugin).
+
+* Create optimized Grok patterns to match the filtered logs to extract attributes.  Avoid using expensive Grok patterns like GREEDYDATA excessively. Ask your account team or SC to help identify any sub-optimal parsing rules.
+
+**GROK Best Practices:**
```

Comment:
just a note that we have quite a few title case uses that we'll want to convert to sentence case. also notice some capitalized words, like 'Log' a few times when not needed.